### ISSUE TITLE
"shift + up arrow" and "shift + down arrow" shows question blocks

### ISSUE THREAD

**Issue by heyycap** [keycastr/keycastr#106]
"shift + up arrow" and "shift + down arrow" shows question blocks. Meanwhile, adding at least one extra modifier, e.g. shift+ctrl+up/down displays correctly.

I'm using a PC keyboard on a Mac.

**Comment by majorcode** [comment]
I get the same question blocks on 100% Apple hardware.

**Comment by mosluce** [comment]
I found the code in KCKeystrokeTansformer.m, line 154 ~ 161

```swift
    if (isShifted && !isCommand)
    {
        NSString *tmp = [@(_keyCode) isEqualToNumber:@48] ? kLeftTabString : keystroke.charactersIgnoringModifiers;
        if (tmp) {
            [mutableResponse appendString:tmp];
            return mutableResponse;
        }
    }
```

keystroke.charactersIgnoringModifiers is always a question blocks.
I comment out these lines, and resolve this issue.

**Comment by mosluce** [comment]
@majorcode 
what's macOS version of your devices?

**Comment by majorcode** [comment]
@mosluce I'm on 10.13.5 now. The machine was on some earlier 10.13.x when I posted my comment.

**Comment by heyycap** [comment]
@mosluce where can I find that file? I originally installed it using the app.zip

**Comment by ins** [comment]
Same issue, apple hardware, "hungarian apple keyboard", macOS 10.13.6

**Comment by NachoAccedo** [comment]
Same issue here. 
Is possible to show Shift + F12 somehow like the command keys?

**Comment by CharlesLiuyx** [comment]
Same bug here, please help to fix this problem.

**Comment by Mark2Mark** [comment]
+1

**Comment by Jotto79** [comment]
Problem still exists

**Comment by akitchen** [comment]
Fixed in v0.9.9
